src/js/core.js

```diff
@@ -945,6 +949,37 @@ function MediumEditor(elements, options) {
                 }
             }
 
+            // If the selection is right at the ending edge of a link, put it outside the anchor tag instead of inside.
```

Comment:
Can all this stuff be moved to the extension instead? Put here just a `if` to test if the extension is enabled and make the call to method in the extension.